src/Controller/Admin/TournamentController.php

```diff
@@ -89,6 +90,19 @@ public function deletePlayerTournamentAction(Request $request, $participantId)
         $participant = $this->getDoctrine()->getRepository(Participant::class)->find($participantId);
         $em = $this->getDoctrine()->getManager();
         $em->remove($participant);
+
+        $tournament = $participant->getTournament();
+        $participants = $em->getRepository(Participant::class)
+            ->getParticipantsByTournamentWhereOrderMoreThanTarget($tournament, $participant->getParticpantOrder());
+
+        /** @var Participant[] $updatedParticipants */
+        for ($i = 0; $i < count($participants); $i++) {
+            $item = $participants[$i];
+            /** @var Participant $item */
+            $currentOrder = $item->getParticpantOrder();
+            $item->setParticipantOrder(--$currentOrder);
```

Comment:
It could be done in one line:
 $item->setParticipantOrder($item->getParticpantOrder() - 1);